### DISCUSSION TITLE
Incorrect url

### DISCUSSION THREAD

**Discussion by Krishnaj20** [Sunbird-inQuiry/Community#4] [Documentation Issues]
Link to telemetry is pointing straight to a specific version on Github. Shouldn't it point to [telemetry.sunbird.org](https://telemetry.sunbird.org/)?

**Comment by Krishnaj20** [comment]
This is now corrected. https://inquiry.sunbird.org/learn/capabilities#e-analytics

**Reply by pramodkvarma** [reply]
Super! Thanks.
